data/service_updates/20240402.yml

```diff
@@ -0,0 +1,21 @@
+---
+title: Manage users within Manage school experience
+summary: |-
+  We've created a new feature that allows you to invite new school experience users directly from within Manage school experience.
+html_content: |-
+  <p>We’ve updated the Manage school experience service
+  so you can now add new users directly on to your school’s account.</p>
```

Comment:
I'd either say now add users or you can add new users. Not sure we need both now and new (if that makes sense).